tests/test_py_counter.py

```diff
@@ -17,12 +17,20 @@ def test_count_comments():
     expected_comments = 14
     assert test_pycount.count_comments() == expected_comments
 
+def test_count_while():
+    """Test case for counting the while loops."""
+    expected_while = 0
+    assert test_pycount.count_while() == expected_while
 
 def test_import_statements():
-    """Test case for counting the import statements."""
+    """Test case for counting import statements."""
     expected_imports = 2
     assert test_pycount.count_import_statements() == expected_imports
 
+def test_count_for():
+    """Test case for counting the for loops."""
+    expected_for = 0
+    assert test_pycount.count_for() == expected_for

```

Comment:
Your test input should contain an example of a `for` loop so that you can test its detection.

Reply:
@RyanHiltyAllegheny